docs/content/en/docs/Recommendations/_index.md

```diff
@@ -6,7 +6,7 @@ weight: 3
 
 Azure Quick Review checks the following recommendations for Azure resources. The recommendations are categorized based on their impact and category:
 
-\#  | Category | Impact | Recommendation | More Info
+#  | Category | Impact | Recommendation | More Info
```

Comment:
add the escape char to avoid issues when rendering the docs with hugo

Reply:
Fixed!